### PR TITLE
Created new publishing public API doc and updated links in old articles

### PR DESCRIPTION
**Pull request by jerry-li-dev**
**Need help with deprecating /docs/publishing-your-api/add-templates/**

**Need help with the order number for the new doc**

- Created new publishing public API doc
- Updated links in older docs to redirect to this doc

### CODE REVIEW

**Comment by jerry-li-dev** [comment]
@SueSmith could you please help me with the order number and having the  /docs/publishing-your-api/add-templates/ links redirect to /docs/publishing-your-api/publish-public-api

**Comment by jerry-li-dev** [comment]
CC: @gitThingsDone

**Comment by SueSmith** [comment]
Sure! So for the redirect:

* If you add it in `redirects.json` in the repo root dir (you can also just delete the file for the page you're removing)

And for the order number:

* It's the `order` field in the frontmatter for the page (at the top in the yaml) and sequential through the whole site (not per folder), and we can do e.g. 107.1, 106.9 etc if you need to insert in between pages or whatever - I would have a play with it and rebuild locally to get a feel for how it works.. 🙂

File: src/pages/docs/publishing-your-api/publish-public-api.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - type: section
+    name: "Next Steps"
+  - type: link
+    name: "Submission guidelines"
+    url:  "/docs/publishing-your-api/api-submission-guidelines/"
+---
+
+You can share your documentation for any [collection](/docs/sending-requests/intro-to-collections/) you publish by adding it to the [Postman public workspaces](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/).
+
+Public workspaces allow you to quickly share your Postman projects with your teammates and others who may be interested in integrating your API technology into their own projects. Adding your collections and documentation to public workspaces allow you share your ideas with the Postman community of over 30 million users.
```
Comment (SueSmith) [inline src/pages/docs/publishing-your-api/publish-public-api.md:24]: I would lose the promotional language here and end the sentence after "community"

File: src/pages/docs/publishing-your-api/publish-public-api.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Your public workspace provides access to the documentation for your API collection and instant access for anyone to import the collection directly into Postman. Community members can access your Postman profile via your workspace overview. From your public profile, users can view other collections, documentation, and workspaces you have already shared.
+
+![Collection Discovery](https://assets.postman.com/postman-docs/public-workspace-overview.jpg)
+
+Collections and public workspaces provide easy to read documentation, requirements, and guidance for people interested in learning and using your APIs. Your document can outline how to use a single endpoint as well as how to integrate with an API as a whole, helping community members learn API skills and discover available services.
```
Comment (SueSmith) [inline src/pages/docs/publishing-your-api/publish-public-api.md:30]: I would lose "easy to read"

File: src/pages/docs/publishing-your-api/publish-public-api.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+![Collection Discovery](https://assets.postman.com/postman-docs/public-workspace-doc.jpg)
+
+## Create your public workspace
+
+Check out steps at [Creating workspaces](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/) to your public workspace to start sharing your collections and documentation.
```
Comment (SueSmith) [inline src/pages/docs/publishing-your-api/publish-public-api.md:36]: I think I'd reword a wee bit here, maybe "Check out how to [create a public workspace](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/) to start sharing...."

**Review by SueSmith — CHANGES_REQUESTED** [review]
looking good @jerry-li-dev! a few minor language suggestions but good to go otherwise imo

**Review by gitThingsDone — CHANGES_REQUESTED** [review]

File: src/pages/docs/publishing-your-api/publish-public-api.md (hunk 1 above)
Comment (jerry-li-dev) [reply src/pages/docs/publishing-your-api/publish-public-api.md:24]: Removed the promotional language.

File: src/pages/docs/publishing-your-api/custom-doc-domains.md [hunk 4]
```diff
@@ -84,4 +84,4 @@ When your domain is verified, you can use it to [publish your API documentation]
 
 ## Next steps
 
-You can make your documentation more discoverable by adding it to the [API Network](/docs/publishing-your-api/add-api-network/) or [Postman Templates](/docs/publishing-your-api/add-templates/).
+You can make your documentation more discoverable by adding it to the [API Network](/docs/publishing-your-api/add-api-network/) or [Postman public workspaces](/docs/publishing-your-api/publish-public-api).
```
[the remark this answers is not in the document]
Comment (jerry-li-dev) [reply src/pages/docs/publishing-your-api/custom-doc-domains.md:87]: - Pending removal of `add-api-network`

File: src/pages/docs/publishing-your-api/publish-public-api.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  - type: section
+    name: "Next Steps"
+  - type: link
+    name: "Submission guidelines"
+    url:  "/docs/publishing-your-api/api-submission-guidelines/"
+---
+
+You can share your documentation for any [collection](/docs/sending-requests/intro-to-collections/) you publish by adding it to the [Postman public workspaces](/docs/collaborating-in-postman/using-workspaces/creating-workspaces/).
+
+Public workspaces allow you to quickly share your Postman projects with your teammates and others who may be interested in integrating your API technology into their own projects. Adding your collections and documentation to public workspaces allow you share your ideas with the Postman community of over 30 million users.
```
[the remark this answers is not in the document]
Comment (jerry-li-dev) [reply src/pages/docs/publishing-your-api/publish-public-api.md:24]: Updated with wording as described in the blog post.

File: src/pages/docs/getting-started/importing-templates.md [hunk 6]
```diff
@@ -14,8 +14,8 @@ contextual_links:
     name: "Creating your first collection"
     url: "/docs/getting-started/creating-the-first-collection/"
   - type: link
-    name: "Publishing templates"
-    url: "/docs/publishing-your-api/add-templates/"
+    name: "Publishing public APIs"
+    url: "/docs/publishing-your-api/publish-public-api"
   - type: link
     name: "Publishing to the API Network"
     url: "/docs/publishing-your-api/add-api-network/"
```
[the remark this answers is not in the document]
Comment (jerry-li-dev) [reply src/pages/docs/getting-started/importing-templates.md:21]: Removed add to api network links and updated submission guidelines.

**Comment by jerry-li-dev** [comment]
Pending updated links to template examples.

File: src/pages/docs/publishing-your-api/api-submission-guidelines.md [hunk 7]
```diff
@@ -42,17 +39,17 @@ The sample code section in your docs will play a significant role in how effecti
...
+Your listing should provide the information developers need to know in order to decide whether they want to try the API or collection out. Try to choose a descriptive name that conveys the purpose of your API without users having to click it to find out more detail. In your description, try to include an overview of what the API / collection does, with enough detail for consumers to figure out whether it's likely to fit their use case. [You can include a name, summary, description, and categories that will determine how your API or collection is listed when users browse public workspaces](/docs/publishing-your-api/publish-public-api)
```
Comment (gitThingsDone) [inline src/pages/docs/publishing-your-api/api-submission-guidelines.md:52]: Can we include steps about how you can add summaries and categories for your public workspaces, APIs, and collections from the profile page. 
we can also update this to something like "use a descriptive name and summary for your public workspaces, APIs, and collections to help consumers develop more context about your APIs, this also helps users get to your public APIs faster through search" 
Categories: Add relevant categories for your public workspaces, APIs, and collections to help your target users find them

File: src/pages/docs/publishing-your-api/api-submission-guidelines.md [hunk 8]
```diff
@@ -42,17 +39,17 @@ The sample code section in your docs will play a significant role in how effecti
...
+If your API requires authentication, make sure you include instructions explaining what developers need to do to acquire and use credentials. Other top level info such as rate limits, content types, or settings that apply across endpoints can appear in your collection description so that they render at the top of your documentation page.
 
 ## Complete your profile
 
-When you publish to the API Network, potential users will initially see your team profile, with APIs listed under it. Taking the time to fill out your team profile gives consumers a clear sense of who a collection is published by—helping them to decide whether to try it out or not. [You can include a team name, description, logo, and custom URL.](/docs/publishing-your-api/add-api-network/#setting-up-your-team-profile)
+When you publish to the API Network, potential users will initially see your team profile, with APIs listed under it. Taking the time to fill out your team profile gives consumers a clear sense of who a collection is published by—helping them to decide whether to try it out or not. [You can include a team name, description, logo, and custom URL.](/docs/getting-started/postman-account/)
```
Comment (gitThingsDone) [inline src/pages/docs/publishing-your-api/api-submission-guidelines.md:46]: Users can also add github and twitter links to their user/team profiles.

**Comment by jerry-li-dev** [comment]
This pr is recreated with https://github.com/postmanlabs/postman-docs/pull/2830 to resolve conflicts

**Comment by jkonrath-postman** [comment]
Closing this without merge, because this PR contains the same changes and will supersede this: https://github.com/postmanlabs/postman-docs/pull/2830
